ntcore/src/main/native/cpp/jni/NetworkTablesJNI.cpp

```diff
@@ -1344,6 +1380,18 @@ Java_edu_wpi_first_networktables_NetworkTablesJNI_isConnected
   return nt::IsConnected(inst);
 }
 
+/*
+ * Class:     edu_wpi_first_networktables_NetworkTablesJNI
+ * Method:    getServerTimeOffset
+ * Signature: (I)Ljava/lang/Object;
```

Comment:
Shouldn't this be `OptionalLong` rather than `Object`?

Reply:
wpiformat forces it to be Object.

Reply:
Weird. Might be a wpiformat bug.
In any case, it's just a comment so I doubt it matters.

Reply:
wpiformat doesn't handle arbitrary types: https://github.com/wpilibsuite/styleguide/blob/main/wpiformat/wpiformat/jni.py#L58

Reply:
According to this old project of mine https://github.com/Starlight220/KoJni/blob/main/src/main/kotlin/io/github/starlight/kojni/JniFunction.kt#L72, it should be the class path:
```kt
        "L" + type.replace('.', '/').trim()
```

Reply:
The JNI generated .h includes the correct thing, but wpiformat forces it to plain Object.  This is just in a comment, it has zero effect on the code.  The problem is wpiformat has no way to determine what the correct signature should be.